github/issues.go

```diff
@@ -44,6 +44,7 @@ type Issue struct {
 	Repository       *Repository       `json:"repository,omitempty"`
 	Reactions        *Reactions        `json:"reactions,omitempty"`
 	Assignees        []*User           `json:"assignees,omitempty"`
+	ClosedBy         *User             `json:"closed_by,omitempty"`
```

Comment:
You might need to either merge the latest master branch into your feature branch, or rebase it on top of latest master.